colorfield/fields.py

```diff
@@ -12,28 +12,51 @@
     from django.utils.translation import ugettext_lazy as _
 
 
-COLOR_RE = re.compile('^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$')
-color_validator = RegexValidator(
-    COLOR_RE, _('Enter a valid color.'), 'invalid')
+COLOR_HEX_RE = re.compile('^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$')
+color_hex_validator = RegexValidator(
+    COLOR_HEX_RE, _('Enter a valid color.'), 'invalid')
+
+COLOR_HEXA_RE = re.compile('#([A-Fa-f0-9]{8}|[A-Fa-f0-9]{4})$')
+color_hexa_validator = RegexValidator(
+    COLOR_HEXA_RE, _('Enter a valid color.'), 'invalid')
```

Comment:
I would improve the error message: `Enter a valid hexa color, eg. #00000088`

Reply:
✔️